source/docs/api-reference/api-usage/actuator-limits.rst

```diff
@@ -0,0 +1,39 @@
+Actuator Limits
+===============
+
+Actuators, such as the CTR-Electronics TalonFX support limits of various kinds. These include limit switches (wired and soft) and supply/stator current limits.
```

Comment:
Maybe something like:
"CTR-Electronics actuators, such as the TalonFX, support various kinds of hardware and software limits."